.github/workflows/publish-latest.yml

```
@@ -3,7 +3,7 @@ name: publish-latest
 on:
   push:
     branches:
-      - master
+      - feature*
```

```suggestion
      - feature/**
```
Pretty sure it will be like this ^^